common/combat_tactics/00_combat_tactics.txt

```diff
@@ -692,23 +686,25 @@ harras_swarm_tactic = {
 				who = horse_archers
 				value = 0.4
 			}
-			OR = {
-				light_troops = { 
-					who = light_cavalry
-					value = 0.3
-				}
-				light_troops = { 
-					who = camel_cavalry
-					value = 0.3
-				}
-			}
+			light_cavalry_troops_030_tactic_trigger = yes
 		}
 	}
+	
+	# horse_archers
+	horse_archers_offensive = 1.2
+	
+	# light_infantry # get modifiers of archers
```

Comment:
I'm confus, why not just combine light_infantry and archers into once commented section?

Reply:
Because there are tactics where archers don't get anything, while light_infantry do.